ci/shell.nix

```diff
@@ -61,6 +69,10 @@ stdenv.mkDerivation ({
     darwin.apple_sdk.frameworks.Metal
     darwin.libobjc
     libiconv
+  ] ++ lib.optionals hardwareTest [
+    (callPackage ./hardware_tests/uhubctl.nix { }) # HACK FIXME replace this with just uhubctl once pinned version of nixpkgs is updated
```

Comment:
Just update to the latest nixpkgs-unstable commit that succeeded in hydra (from https://status.nixos.org) so we can drop this hack

Reply:
Tried that but it breaks tests: https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/1131566999 https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/1131567000 will investigate.

Reply:
So I was only able to drop the hack by introducing another hack in c76aee0. Without diving into tox i don't know how to create a minimal reproducer, not even 100% sure if it's a bug in nixpkgs or tox.